package.json

```diff
@@ -79,7 +83,6 @@
     "babel-preset-stage-0": "^6.24.1",
     "babel-register": "^6.26.0",
     "babili-webpack-plugin": "^0.1.2",
-    "bootstrap": "^4.3.1",
```

Comment:
What is the intent of removing bootstrap here?

Reply:
to get rid of a duplicate dependency warning. I believe developer dependencies are the additional packages needed for development. Since "bootstrap" is in the normal app dependencies, it does not need to be listed in the developer dependencies.

Reply:
Ah, gotcha. Thanks for the clarification